### PR TITLE
fix SF 4.2 compatibility

### LINKED ISSUE #34: Incompatible symfony/validator release - v4.2
https://github.com/symfony/validator/commit/0e2b304fd0ffbd7d236ef9df4f58d4017152934f#diff-7314b7f53f439cdf2ede17404f443db0

Because there is missing typehint for `$translator` - causes error for DI Container Builder.
I think, that set default value in extension fix this.

### PR DESCRIPTION
**Pull request by janatjak**
Resolve #34

### CODE REVIEW

File: src/DI/ValidatorExtension.php [hunk 1]
```diff
@@ -73,9 +73,15 @@ public function loadConfiguration()
 			->setClass('Symfony\Component\Validator\ConstraintValidatorFactoryInterface')
 			->setFactory('Kdyby\Validator\ConstraintValidatorFactory');
 
+		// allow use new SF 4.2 Contract component
+		$contextFactoryParams = ['translationDomain' => $config['translationDomain']];
+		if (interface_exists('\Symfony\Contracts\Translation\TranslatorInterface')) {
```
Comment (enumag) [inline src/DI/ValidatorExtension.php]:
```suggestion
		if (interface_exists('Symfony\Contracts\Translation\TranslatorInterface')) {
```

Comment (janatjak) [reply src/DI/ValidatorExtension.php]: Ok, I replace it by `Symfony\Component\Translation\TranslatorInterface`

**Comment by janatjak** [comment]
Tests passed, something else?

**Comment by enumag** [comment]
It's fine in my opinion.

ping @Majkl578

**Pull request merged**